docs/ops/sort/NMSRotated_13.md

```diff
@@ -0,0 +1,142 @@
+# NMSRotated {#openvino_docs_ops_sort_NMSRotated_13}
+
+@sphinxdirective
+
+.. meta::
+  :description: Learn about NMSRotated-13 - a sorting and maximization
+                operation, which requires five input tensors.
+
+**Versioned name**: *NMSRotated-13*
+
+**Category**: *Sorting and maximization*
+
+**Short description**: *NMSRotated* performs non maximum suppression of the rotated boxes with predicted scores.
+
+**Detailed description**: *NMSRotated* performs regular non maximum suppression, but the value of IoU is calculated for bounding boxes rotated by corresponding angle.
+
+The general algorithm is described below:
+
+1.  Let ``B = [b_0,...,b_n]`` be the list of initial detection boxes, ``S = [s_0,...,s_N]`` be  the list of corresponding scores.
+2.  Let ``D = []`` be an initial collection of resulting boxes.
+3.  If ``B`` is empty then go to step 8.
+4.  Take the box with highest score. Suppose that it is the box ``b`` with the score ``s``.
+5.  Delete ``b`` from ``B``.
+6.  If the score ``s`` is greater or equal than ``score_threshold``  then add ``b`` to ``D`` else go to step 8.
+7.  For each input box ``b_i`` from ``B`` and the corresponding score ``s_i``, set ``s_i = s_i * func(rotated_iou(b_i, b))`` and go to step 3.
+8.  Return ``D``, a collection of the corresponding scores ``S``, and the number of elements in ``D``.
+
+Here ``func(iou) = 1 if iou <= iou_threshold else 0``.
```

Comment:
Perhaps `func(rotated_iou(b_i, b))`?
Could you add explanation for `rotated_iou` as well?

Reply:
Done